#166313415 create roles model

JuwonAbiola opened the pull request:
#### What does this PR do?
Create roles model
#### Description of Task to be completed?
-create roles model
-create admin roles seeder file
-create patron roles seeder file
#### How should this be manually tested?
clone or download the project
checkout to ft=create-roles-model-166313415
git pull to get the latest update
npm i to install dependencies
create a database
run npm run db:roll to create table
run npm run db:seed to test the feature
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166313415](https://www.pivotaltracker.com/n/projects/2350588/stories/166313415)

#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

allebd commented:
Your commit PR title is not using the right convention specified

Adekoreday commented on server/database/seeders/20190529165259-patron.js:
```diff
@@ -0,0 +1,10 @@
+    updatedAt: new Date()
+  }], {}),
+
+  down: (queryInterface, Sequelize) => queryInterface.bulkDelete('roles', null, {})
```
the task is meant to create models alone do not include migration and seed files

Adekoreday commented on server/database/seeders/20190529165259-patron.js:
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('roles', [{
```
you can add eslint-disable above each line you are using fat arrow functions to prevent hounCI from flagging error

Adekoreday reviewed (changes requested):
you need to add roles associations there exist a relation between roles and userRoles in your role model

Adekoreday reviewed (changes requested):
change the typre from ft in your commit message

allebd reviewed (commented):
Join your seeders file together as an array, you should have one seed file, not two

JuwonAbiola commented:
> Join your seeders file together as an array, you should have one seed file, not two

I have just one seeder file

Adekoreday reviewed (changes requested):
the above is on develop

Lundii reviewed (changes requested):
* You should add `/* eslint-disable */ ` in all your migrations, models and seeders file as started by the TTL

Lundii commented:
> ![image](https://user-images.githubusercontent.com/44194416/58644711-bfeda180-82f9-11e9-9ca7-e37061c75b16.png)
> 
> the above is on develop
> 
> the one below is on your branch it might cause some conflicts i think you need to resolve this
> ![image](https://user-images.githubusercontent.com/44194416/58644924-38546280-82fa-11e9-84b9-e4530ce22cee.png)

Korede, please fetch his branch again. There is just one  seeders file

JuwonAbiola committed:
166313415-story (model): roles model

-create roles model
-create admin seeder file
-create patron seeder file

[Finishes #166313415]
